darwin/verify_icecream.py

```diff
@@ -57,6 +62,13 @@ def verify_icecream():
         if not("inet " in ifconfig and "inet6" in ifconfig):
             continue
 
+        # Now check for speed in case of bad physical ports
+        if not("1000baseT" in ifconfig):
```

Comment:
Maybe we should only look for this, rather than a whitelist of adapters.

Reply:
If we use the whitelist, we call ifconfig once. If we check on speed, we will have to call ifconfig N times. Does the speed of the check matter?